Alright, it seems that Testcontainers was able to detect your Docker and successfully connected to it.

Select your problem:

| |
|-|
|[Unable connect to my running container](past_checks/connection_issues.md)|
|[Other](todo.md)|
